src/index.html

```diff
@@ -5,10 +5,72 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/reset.css">
+    <link rel="stylesheet" href="styles/style.css">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <div class="header__logo">
+        <a class="header__logo-link" href="#home">
+          <img src="images/logo.png" alt="moyo logo">
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="list">
+          <li class="list__items">
+            <a href="#apple" class="items__link is-active">
```

Comment:
```suggestion
            <a href="#apple" class="nav__link is-active">
```

it is not critical for now (you didn't learn BEM yet), but it is the best practice to name the block and its elements